### PR TITLE
update Chrome es2017 support

### PR DESCRIPTION
**Pull request by chicoxyzzy**
`Proxy "ownKeys" handler, duplicate keys for non-extensible targets` is marked as supported

### CODE REVIEW

**Comment by chicoxyzzy** [comment]
I've also marked Chrome 51 as stable (and 50 as obsolete)

**Comment by chicoxyzzy** [comment]
rebased to resolve conflicts

File: data-es2016plus.js [hunk 1]
```diff
@@ -853,6 +853,8 @@ exports.tests = [
      return Object.getOwnPropertyNames(P) + '' === "a,a,b,b";
      */},
     res: {
+      chrome51: true,
+      chrome52: true,
```
Comment (mgol) [inline data-es2016plus.js]: Setting `chrome51: true` should be enough, shouldn't it?

Comment (chicoxyzzy) [inline data-es2016plus.js]: yep. fixed. thank you

**Pull request merged**